docs/catalog/cloud/instal-supported-add-ons-and-extensions.md

```diff
@@ -0,0 +1,34 @@
+---
+version: 1.0
+submitted_by: yelghali
+published_date: tbd
+category: cloud
+tags: 
+- compute
+- kubernetes
+- role:cloud-engineer
+- size:medium
+---
+
+# Install supported add-ons and extensions
+
+## Description
+Add-ons and extensions covered by the editor support policy provide additional and supported functionality, while allowing you to benefit from the latest performance improvements and energy optimizations throughout your service lifecycle.
+
+## Solution
+
+Use supported Add-ons and extensions to benefit from newer and more efficient services as they become available.
```

Comment:
Similar to comments in PR https://github.com/Green-Software-Foundation/green-software-patterns/pull/154, newer doesn't always mean more efficient. Is there a metric or additional specificity you can provide in the solution area to determine if the add-on or extension has a positive impact to your emissions?